Update _index.md | make it clearer that facets value include restricted tests (#27399)
DataDog/documentation: content/en/synthetics/explore/_index.md

| `Notification` | The handle used by the Synthetic test for notifications.                      |
| `Env`          | The environment the Synthetic test is running on.                             |
| `CI/CD Execution Rule` | The status of the test run: `Blocking`, `Non-blocking`, or `Skipped`. |

> | `Endpoint`     | The url or domain the test is targetting                                      |

Reviewer: ```suggestion
| `Endpoint`     | The url or domain the test is targeting.                                      |
```

> <div class="alert alert-info"> You might see values fof the facets coming from tests you can't access </div>

Reviewer: ```suggestion
<div class="alert alert-info"> Note: You might see facet values originating from tests you don't have access to. </div>
```
